Using-Mastodon/FAQ.md

```diff
@@ -64,50 +71,114 @@ Try [TootyFruity](https://play.google.com/store/apps/details?id=ch.kevinegli.too
+
+#### How do I prevent people from impersonating me?
+Because anyone can create a username on any instance, it is impossible to prevent others from using the same username on another instance. Some people have taken to running their own single-user instance to verify themselves, or use [Keybase](https://keybase.io/).
+
+#### How do I enable Two Factor Authentication?
+If your instance supports it, the option will be in Preferences under Two-factor Authentication. [Mastodon's 2FA](2FA.md) uses both a QR code and plain-text secrets.
+
+#### Why can't I see someone's toots?
+This could occur for several reasons. A person's posts may be private. You must be a follower of a private account to see their non-public posts. Additionally, if someone has blocked you, you will no longer see their non-public posts.
+
 #### What’s trending?
-Follow the esteemed [@TrendingBot@mastodon.social](https://mastodon.social/@TrendingBot).
+To find out what’s trending, follow the esteemed [@TrendingBot@mastodon.social](https://mastodon.social/@TrendingBot).
 
 #### What’s “CW”?
 “CW” stands for “Content Warning.” You can use it to hide your toot, like a spoiler.
 
+#### How does the “Mark Image as Sensitive” feature work (“NSFW”)?
+When you add a picture to your post using the camera icon, an additional toggle will appear [typically as "NSFW"]. Clicking this will hide your image behind a "Sensitive Content" warning, preventing others from seeing your images until they click on them.
+
 #### What’s with the pineapples?
 They’re tasty, and they make people smile. Just go with it.
 
+#### I see “Awoo” a lot, what does that mean?
+Try saying it out loud. It’s fun!
+
 #### How do I delete my account?
 At the moment, you will have to toot the admin of your instance for help on that one.
 
+<a name="who-is-my-admin-and-how-do-i-contact-follow-them"></a>
 #### Who is my admin, and how do I contact/follow them?
 Click on **Getting Started**, then **Extended Information**. An info page displays. Hopefully, they put their contact information in there!
 
-![Getting Started](https://octodon.social/media/TKlZpSNRRcHBOEFNlGs) ![Extended Information](https://octodon.social/media/D7bfl1NYzNy4XIT8nLs) ![Info Page](https://octodon.social/media/zvoI8yUt4uurZcKqFRk)
+<img src="screenshots/toolbar-getting_started.png" alt="Getting Started Button" height="200"/>
+<img src="screenshots/getting_started-extended_information.png" alt="Extended Information" height="200"/>
+<img src="screenshots/admin_info.png" alt="Admin Info" height="200"/>
 
 #### Wow, I keep seeing offensive stuff from a particular instance, is there a way to block *all* of it?
 Not without a lot of clicking, sorry. But, your instance admin can do it! Please contact her.
 
+#### I blocked someone, why can I still see their posts?
+Blocking doesn’t catch when toots are boosted.
```

Comment:
I believe the question was from a while ago, so I'll remove it since it probably won't come up.